admin_manual/installation/example_openbsd.rst

```diff
@@ -0,0 +1,236 @@
+.. _openbsd_installation_label:
+
+Nextcloud does not have official OpenBSD or other BSDs support
```

Comment:
![image](https://user-images.githubusercontent.com/3902676/111221465-cfc39c00-85da-11eb-9d01-77dd14218e7a.png)

> Nextcloud does not have official OpenBSD or other BSDs support

Is now a navigation item. Can you use the box again?

Reply:
```
.. _openbsd_installation_label:

Example installation on OpenBSD
===============================

.. warning::

    Nextcloud does not have official OpenBSD or other BSDs support

In this install tutorial we will be deploying Nextcloud on a minimal OpenBSD with our own httpd(8), PHP, PostgreSQL and redis (for -stable or -current are the same steps).
```

![image](https://user-images.githubusercontent.com/3902676/111221917-6abc7600-85db-11eb-8afe-a8298bde3d14.png)